configure_combinations

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+sh build_posix/reconf
```

Comment:
This line assumes the script will be called under top-level directory of wiredtiger tree. I'd prefer making it portable by querying out and cd-ing to the top-level directory.

Reply:
done